test/e2e/mpi_job_test.go

```diff
@@ -230,6 +217,98 @@ var _ = ginkgo.Describe("MPIJob", func() {
 		})
 
 	})
+
+	ginkgo.Context("with scheduler-plugins", func() {
```

Comment:
can we only install the custom scheduler at this point?

Reply:
Does that mean uninstalling the default scheduler?

Reply:
No. I mean not to install the custom scheduler for the whole suite, just this `ginkgo.Context`.

Not that scheduler-plugins is currently problematic, but in general is better to isolate dependencies from tests that don't need it.

Reply:
It makes sense. I'll try it.